doc/specifications/rest/REST-API-V2.rst

```diff
@@ -210,6 +210,242 @@ SSL Client Authentication
 
 The REST API provides authenticating a user by a subject in a client certificate delivered by the web server configured as SSL endpoint.
 
+Bookmark
+========
+
+Overview
+--------
+
+====================== ================ ================ =============================== ================ ================
+Resource                     POST             GET              HEAD                      PATCH/PUT        DELETE
+---------------------- ---------------- ---------------- ------------------------------- ---------------- ----------------
+/bookmark              .                List bookmarks   .                               .                .
+/bookmark/<path>       Create bookmark  .                Check if location is bookmarked .                Delete bookmark
+====================== ================ ================ =============================== ================ ================
+
+Specification
+-------------
+
+List bookmarks
+~~~~~~~~~~~~~~
+
+:Resource: /bookmark
+:Method: GET
+:Description: List bookmarked locations
```

Comment:
Does it load the bookmarks of the current user ? I assume that it does, and if it does, we should clarify it in the description. The same applies to the other resources below (I don't think that the "personal" aspect of this is mentioned anywhere in the spec).